modules/CmpdReg/src/server/routes/CmpdRegRoutes.coffee

```diff
@@ -147,47 +116,6 @@ exports.getAuthorizedCmpdRegProjects = (req, resp) ->
 		resp.status "200"
 		resp.end JSON.stringify allowedUserProjects
 
-exports.saveProjects = (jsonBody, callback) ->
```

Comment:
No longer being used since we migrated all of these to thing projects or passthrough.